pkgs/by-name/ey/eyedropper/package.nix

```diff
@@ -1,36 +1,37 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-, rustPlatform
-, cargo
-, pkg-config
-, meson
-, ninja
-, blueprint-compiler
-, glib
-, gtk4
-, libadwaita
-, rustc
-, wrapGAppsHook4
-, appstream-glib
-, desktop-file-utils
+{
+  lib,
```

Comment:
Reformatting should be a separate commit. LGTM otherwise.

Reply:
Ignored as the diff isn't too big